package.json

```diff
@@ -98,12 +90,12 @@
     "test:watch": "jest --config=jest.config.json --watch",
     "test:page": "webpack-dev-server --open --config test/page/webpack.config.cjs",
     "watch": "nodemon --watch src --exec 'npm run build'",
-    "inject-version": "cross-var replace \"@VERSION@\" $npm_package_version dist/api.js",
```

Comment:
This is used to display the swell-js version in the Wappalyzer browser extension. Does it work differently now?

Reply:
That's right, it works differently now: https://github.com/swellstores/swell-js/blob/c82c5d9a951ae1931fe11ea9d78b5c6ba32421f6/rollup.config.js#L16

By the way, we do not have such a file as `dist/api.js`.